components/JournalyEditor/helpers.ts

```diff
@@ -140,7 +140,8 @@ const wrapLink = (editor: Editor, url: string) => {
 }
 
 const toggleLink = ({ editor, t }: ToggleArgs) => {
-  let url: string | undefined | null = window.prompt('Enter the URL of the link:')
+  const enterUrlPrompt = t ? t('enterUrlPrompt') : 'Enter the URL of the link:'
```

Comment:
hmm, what is the situation where `t` would be falsy?

Reply:
Yeah I did this for now to satisfy TS because `t` was set as optional when created, but I agree that if things are set up properly it shouldn't ever be falsy. Shall we just make it required so we can remove this check?

Reply:
Yeah, I think making it required in `ToggleByTypeArgs` makes sense. I'm guessing it was marked optional for a reason so we may have to move a null check up to where we call into `toogleByType` (heh, just noticed that misspelling) but we almost certainly should be checking it down here in these toggle functions